docs/web3-eth-accounts.rst

```diff
@@ -163,7 +163,7 @@ Returns
     - ``s`` - ``String``: Next 32 bytes of the signature
     - ``v`` - ``String``: Recovery value + 27
     - ``rawTransaction`` - ``String``: The RLP encoded transaction, ready to be send using :ref:`web3.eth.sendSignedTransaction <eth-sendsignedtransaction>`.
-
+    - ``txHash`` - ``String``: The transaction hash for the RLP encoded transaction.
```

Comment:
I think we should use **transactionHash** instead of **txHash** to make module consistent. what do you say @nivida?

Reply:
I agree. Let's use `transactionHash` then.